src/util.js

```diff
@@ -60,24 +60,46 @@ module.exports = {
 
     return urlSafeB64Str;
   },
-  decode64: function(urlSafeB64Str) {
+  decode64: function (urlSafeB64Str) {
     var b64Str = urlSafeB64Str.replace(/-/g, '+').replace(/_/g, '/'),
       encodedUtf8Str = atob(b64Str),
       str = decodeURIComponent(escape(encodedUtf8Str));
 
     return str;
   },
-  domReady: function(cb) {
+  domReady: function (cb) {
     if (document.readyState === 'complete') {
       setTimeout(cb, 0);
     } else if (document.addEventListener) {
       document.addEventListener('DOMContentLoaded', cb, false);
     } else {
-      document.attachEvent('onreadystatechange', function() {
+      document.attachEvent('onreadystatechange', function () {
         if (document.readyState === 'complete') {
           cb();
         }
       });
     }
-  }
+  },
+  getMetaTagValue: function (propertyName) {
```

Comment:
I like this refactor, it makes a lot more sense to have this utility located here, nice work 👏